core/json/src/test/scala/net/liftweb/json/ExtractionBugs.scala

```diff
@@ -59,6 +59,11 @@ object ExtractionBugs extends Specification {
     json.extract[Response] mustEqual Response(List(Map("one" -> 1, "two" -> 2)))
   }
 
+  "Extracting List[Option[String]]" in {
```

Comment:
Phrasing here seems wrong (“Extraction should handle…” seems to be the trend above).